sql/resourceloaderarticlesPriorityMigration.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE /*_*/resourceloaderarticles
+  ADD COLUMN IF NOT EXISTS `rla_priority` int(11) NOT NULL;
```

Comment:
```suggestion
  ADD COLUMN IF NOT EXISTS `rla_priority` int(11) NOT NULL;
 UPDATE /*_*/resourceloaderarticles SET `rla_priority` = 100;
```
What do we want the default value to be? I set 100 here, but we should make a decision.

Reply:
This is to set a value for the already existing rows in the table.

Reply:
that's also fine, but I'd like to have it in the migration explicitly